cmd/ck8s/k8s.go

```diff
@@ -129,6 +139,13 @@ func addNode(
 			),
 		}
 	}
+	if viper.IsSet(diskSizeFlag) {
+		providerSettings = map[string]interface{}{
```

Comment:
You will need to move the provider settings map outside of this scope so that it can be shared with "es-local-storage". Right now you can only set one or the other.